README.md

```diff
@@ -13,6 +13,14 @@ Add the dependency, as usual:
 ewasm-api = "0.9"
 ```
 
+Or enable specific features:
```

Comment:
I would move this to right below the list of feature sets. It also occurs to me that the organization of the README mixes build and usage instructions, which should probably be fixed at some point.